packages/block-editor/src/components/panel-color-settings/README.md

```diff
@@ -0,0 +1,28 @@
+# `PanelColorSettings`
+
+Render a panel with color choices shown in circles, allowing to user to pick up them.
+
+## Properties
+
+### `children`
+
+* **Type:** ``
+* *Required.* ...
+
+...
+
+### `colorSettings`
+
+* **Type:** `Array`
+* *Required.* ...
+
+An array of props used for mapping both ColorIndicator and ColorPaletteControl components.
```

Comment:
> used for mapping

Without the context of the code I wasn't 100% sure what this meant. Could we elaborate slightly?

Reply:
While I was learning the code just simply documenting it. We can create a new branch and cherry-pick it up if we want. Zero objections.